lib/docs/token-response-types.md

```diff
@@ -0,0 +1,74 @@
+# Token Response Types
```

Comment:
Instead of creating a docs folder in /lib can we put this in /lib/msal-core if this is going to be core specific or in /docs if we want to have a top level folder for docs that apply to all libraries

Reply:
Agree, I would put this in `lib/msal-core/docs` to start.

Reply:
Yes, good call, I'll move it!